packages/core/src/types.ts

```diff
@@ -1325,3 +1328,23 @@ export type InterpreterFrom<
 >
   ? Interpreter<TContext, TStateSchema, TEvent, TTypestate>
   : never;
+
+export interface ActorContext {
+  parent?: ActorRef<any, any>;
+  self: ActorRef<any, any>;
```

Comment:
it should be easy to type `self` more strictly - just add existing~ generics to `ActorContext`